build-support/virtualenv

```diff
@@ -9,13 +9,18 @@ VIRTUALENV_PACKAGE_LOCATION=${VIRTUALENV_PACKAGE_LOCATION:-https://pypi.io/packa
 REPO_ROOT=$(cd $(dirname "${BASH_SOURCE[0]}") && git rev-parse --show-toplevel)
 source ${REPO_ROOT}/build-support/common.sh
 
+# Note the caller may set $PY to whichever Python version
+# they want. If not set, we default to Python 2.7.
 if [[ -z "${PY}" ]]; then
-  python_bin_name="python2.7"; [ "${PANTS_USE_PYTHON3}" = true ] && python_bin_name="python3"
-  if which "${python_bin_name}" >/dev/null; then
-    PY=`which ${python_bin_name}`
-  else
-    die "No ${python_bin_name} interpreter found on the path. Python will not work!"
-  fi
+  PY="python2.7"
+fi
+
+# Locate the binary's path to log to user or
+# die gracefully if not found.
```

Comment:
Hm this comment should be on one line. Looked different locally.

I'll fix if there are any other changes I need to make.